### ISSUE TITLE
Fps Stuck

### ISSUE THREAD

**Issue by redxell** [Whitetigerswt/gtasa_crashfix#40]
I used to have 97 fps while connecting to server then 70-80 fps in interior and fps ranging from 50-40 outside, After installing this my fps just got stuck in 50 neither it goes up and neither down, Is there anything which could help?

**Comment by Whitetigerswt** [comment]
That's weird, is your frame limiter off?

**Comment by redxell** [comment]
Yes Of course it is, but if i turn it on My fps start dropping or increasing from 50.

**Comment by xJubz** [comment]
I used to have a similar problem. Have you tried updating your drivers? I went through & made sure they were all up-to-date and it seemed to solve it for me.

**Comment by redxell** [comment]
Yes the first thing I've tried is too make sure that my drivers are up to date, However uninstalling the .exe version and using .asi version have solved the problem but still I haven't got satisfied results it's like 282 fps on startup then it's get down to 50 -40 fps never increases from it.
